sites/fast-website/src/app/components/content-card/content-card.template.ts

```diff
@@ -0,0 +1,23 @@
+import { html, when } from "@microsoft/fast-element";
+import { ContentCard } from "./content-card";
+
+export const ContentCardTemplate = html<ContentCard>`<fast-card>
```

Comment:
This is really only "card-like" on hover and I think we'll have a competing "content card" potentially based on the design. Can we find another name that might work here?

Reply:
would framework-and-community-content be more suitable?

Reply:
So for this template FrameworkAndCommunityContentTemplate.

Reply:
Let's go with something like `content-block` or `content-placement` or `content-_____` to try and keep it less verbose and less focused on the sections. If we add additional sections that name would not extend to support it